manuscript/03.results.md

```diff
@@ -1,14 +1,14 @@
 
-These observations reveal that population-level drug response measurements provide an average of dynamical behaviors that mask variation existing in a population of cells over time. Measurements that record the history of cells (e.g., CFSE staining, Luria-Delbruck experiment) can help to identify the various dynamics within cell populations but must make assumptions about the dynamics of heterogeneity [@DOI:10.1073/pnas.1715639115; @PMID:28607484]. Lineage measurements, by contrast, provide sufficiently rich measurements to quantify the specific structure of heterogeneous response. By observing individual cells, a variety of phenotypic measurements such as inter-mitotic times, cell cycle phase durations, motility, cell death and division, morphology, and protein and transcription factor levels can be characterized in parallel [@doi:10.1126/science.abc1944].
+As a general overview of the data, we randomly sampled lineages from the gemcitabine treated cells (Fig. [@fig:lineage]c). Gemcitabine is a chemotherapy agent that disrupts DNA replication and results in extension of and apoptosis in S phase [@PMID:8431926]. The S/G2 phase lengths in treated cells are noticably extended compared to control cells, and cell division is diminished. Individual lineages include anywhere from zero to 3 cell divisions and show tightly shared dynamic behavior among cells and their close relatives. By observing individual cells, a variety of phenotypic measurements such as inter-mitotic times, cell cycle phase durations, motility, cell death and division, morphology, and protein and transcription factor levels can be characterized in parallel [@doi:10.1126/science.abc1944].
 
-![**Population-based measurements are insufficient to distinguish the structure of heterogeneous populations.** Randomly selected lineages of AU565 cells (a) untreated or (b) treated with 5 nM gemcitabine. Each line indicates the lifetime of one cell. The line branching into two lines indicates cell division. The G1 and S/G2 phase durations are shown in solid thick and thin lines, respectively.](figure1.svg){#fig:lineage}
+![**Population-based measurements are insufficient to distinguish the structure of heterogeneous populations.** Selected lineages of AU565 cells (a) heterogeneous untreated (b) homogeneous untreated lineages. (c) Randomly selected lineages of AU565 cells treated with 5 nM gemcitabine. Each line indicates the lifetime of one cell. The line branching into two lines indicates cell division. The G1 and S/G2 phase durations are shown in solid thick and thin lines, respectively.](figure1.svg){#fig:lineage}
```

Comment:
What does "heterogeneous untreated" and "homogeneous untreated" mean?

Reply:
I meant figure 1a includes single lineages, as well as lineages with 4 divisions, but in figure 1b cells mostly divide at the same rate and include 2-3 generations.. I will replace the sentence with something better.

Reply:
I replaced it with:
"(a) untreated lineages with high growth dynamic variations and (b) untreated lineages with similar growth dynamics."